src/linux/system.rs

```diff
@@ -844,6 +872,10 @@ DISTRIB_DESCRIPTION="Ubuntu 20.10"
             get_system_info_linux(InfoType::Name, &tmp1, Path::new("")),
             Some("Ubuntu".to_owned())
         );
+        assert_eq!(
+            get_system_info_linux(InfoType::DistributionID, &tmp1, Path::new("")),
+            Some("ubuntu".to_owned())
```

Comment:
Won't it be `Ubuntu` here based on the `DISTRIB_ID` value?

Reply:
No, `tmp1` has `ID=ubuntu`.

This is by the way the reason I would like to avoid lsb-release altogether, it's not consistent with os-release.

Reply:
Oh you're right. Sorry didn't scroll up enough.